pkg/config/config.go

```diff
@@ -247,12 +248,12 @@ func (env *TestEnvironment) ResetOc() {
 
 func (env *TestEnvironment) doAutodiscover() {
 	log.Debug("start auto discovery")
-	if len(env.Config.TargetNameSpaces) != 1 {
-		log.Fatal("a single namespace should be specified in config file")
+	for _, ns := range env.Config.TargetNameSpaces {
+		env.NameSpacesUnderTest = append(env.NameSpacesUnderTest, ns.Name)
```

Comment:
add 	env.NameSpacesUnderTest=nil in reset(), otherwise autodiscovery will keep adding the same namespace each time this is called. See below:
func (env *TestEnvironment) reset() {
	log.Debug("clean up environment Test structure")
	env.ResetOc()
	env.Config.Partner = configsections.TestPartner{}
	env.Config.TestTarget = configsections.TestTarget{}
	env.TestOrchestrator = nil
	env.NameSpacesUnderTest=nil
This should fix the smoke test

Reply:
fixed